Test.Activities.Automation.ActivityLib/Models/Activity/Classes/Member.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+
+namespace Test.Activities.Automation.ActivityLib.Models.Activity.Classes
+{
+    public class Member
+    {
+        public int? MentorId { get; set; }
```

Comment:
Instead of nullable `MentorId` and `RootMentorId` properties, just define one nonnullable `UserId`